site/content/en/docs/installation/_index.md

```diff
@@ -205,3 +205,6 @@ To uninstall Kueue, run the following command:
 ```sh
 make undeploy
 ```
+## Install Kueue with [Helm](https://helm.sh/)
+
+To install and configure Kueue with Helm, follow the [instructions](/charts/kueue/README.md).
```

Comment:
This link won't work.

I think we should do it the other way:
change the README.md file to refer to a page in the website.

Reply:
We can refer to https://github.com/kubernetes-sigs/kueue/blob/main/charts/kueue/README.md directly.

Reply:
Can you add an indexer in the beginning?